receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/resource_test.go

```diff
@@ -33,7 +64,7 @@ func TestContainerResource(t *testing.T) {
 		CreatedAt:     "2020-07-30T22:12:29.837074927Z",
 		StartedAt:     "2020-07-30T22:12:31.153459485Z",
 		FinishedAt:    "2020-07-31T22:12:29.837074927Z",
-		KnownStatus:   "RUNNING",
+		KnownStatus:   "STOPPED",
 		ExitCode:      &exitCode,
```

Comment:
For exit code, as we discussed, can we add two unit tests. One with exitCode and the other without exitCode?

Reply:
Same for `FinishedTime`.

Reply:
saw you added them. Thanks.